<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>426</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnRead">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>方正兰亭超细黑简体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>读取文件</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSort">
       <property name="font">
        <font>
         <family>方正兰亭超细黑简体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>排序</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSearch">
       <property name="font">
        <font>
         <family>方正兰亭超细黑简体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>检索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnRenew">
       <property name="font">
        <font>
         <family>方正兰亭超细黑简体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>电子地图信息管理系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelWelcome">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      欢迎使用电子地图信息管理系统
            请选择要使用的功能！</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>961</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="SortChoose" name="SortWidget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>751</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="ReadFile" name="ReadWidget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="SearchUi" name="SearchWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="RenewWidget" name="RenewWidge" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SortChoose</class>
   <extends>QWidget</extends>
   <header location="global">sortchoose.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReadFile</class>
   <extends>QWidget</extends>
   <header location="global">readfile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchUi</class>
   <extends>QWidget</extends>
   <header location="global">searchui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenewWidget</class>
   <extends>QWidget</extends>
   <header location="global">renewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
